pkg/az/group.go

```diff
@@ -14,9 +14,10 @@ var (
 
 // GroupCommand ensures that a group exist or not
 type GroupCommand struct {
-	action   string
-	Name     string `yaml:"name"`
-	Location string `yaml:"location"`
+	action      string
+	Description string `yaml:"description"`
```

Comment:
I'm wondering why you added Description here? Is it used by anything?

Reply:
I wanted to make sure that all command has the same properties. If it's not necessary, I can remove it

Reply:
Nope, it's fine I just wasn't sure why you added it. Consistency is good! 👍